src/components/HURUmap/LocationHeader/index.stories.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import { ReactComponent as Print } from "@/pesayetu/assets/icons/print.svg";
+import LocationHeader from "@/pesayetu/components/HURUmap/LocationHeader";
+
+export default {
+  title: "Components/HURUmap/LocationHeader",
+  argTypes: {},
+};
+
+const Template = ({ ...args }) => <LocationHeader {...args} />;
```

Comment:
```suggestion
const Template = (args) => <LocationHeader {...args} />;
```
Destructure ( {some, ...rest}) only when you need to pick some props and not others.